runtime/lua/vim/lsp/util.lua

```diff
@@ -2323,6 +2323,72 @@ function M._refresh(method, opts)
   end
 end
 
+---@class (private) vim.lsp.util.enable.Filter
```

Comment:
I would renaming this to `vim.lsp.enable.Filter` and then removing `vim.lsp.semantic_tokens.enable.Filter` to use this type instead.